digitizer/parse.py

```diff
@@ -67,6 +67,9 @@ def prepare_isotherm_dict(form):
     if form.inp_tabular.value:
         data['tabular_data'] = True
     data['digitizer'] = form.inp_digitizer.value
+    data['associated_content'] = [form.inp_figure_image.filename]
```

Comment:
Just one thing to keep in mind: as things are set up now, this will not be the filename in the "package" - there, I'm prepending the isotherm name, e.g. `<doi>.Isotherm1_figure1.png`
https://github.com/ltalirz/isotherm-digitizer-panel/blob/abbbc5831a46aa8e4edfca5aa0b8ae73d9143b29/digitizer/submission.py#L102-L105

The value in the raw JSON can still be the original one, since the actual filename is easily computed.

Reply:
Good point, I should have caught that. I'll revisit this feature and make a correction.

Reply:
Up to you. As mentioned, I think it's ok to record just the original filename

Reply:
Revised solution: filename stays "as is" unless the submission ZIP archive is created. in that case, the 'associated_content' key is updated with the new filename(s). Update in PR #43